santorini.game.php

```diff
@@ -679,6 +689,10 @@ public function announceWin($playerId, $win = true)
         'player_name' => $players[0]->getName(),
       ]);
     }
+    // endgame: reveal all secret info 
+    $secrets = $this->board->getSecretPieces();
+    foreach ($secrets as $secret)
+      $this->notifyAllPlayers('revealPiece', '', ['piece' => $secret]); 
```

Comment:
For Tartarus, this shows the abyss but does not update the UI. Not sure what the best way is to do it. Maybe move the pieces to the board then call the relevant power::updateUI()?

Also, what does the *Instant notification do? Is it more appropriate to call revealPieceInstant rather than revealPiece here?

Reply:
Normal notification enforces a delay in the UI while the animation runs. It's maybe 1 or 2 seconds? In the main game .is file in setupNotifications function. The "instant" version doesn't delay the UI.

Three pieces in a loop without instant: revealed one by one, you can't see the second piece reveal until after the first piece completed the animation.